src/desktop/analytics/criteo.js

```diff
@@ -50,63 +50,6 @@ if (pageType === "auctions") {
       )
     })
   }
-} else if (pageType === "artwork" && !pathSplit[3]) {
```

Comment:
With the removal of the old artwork app, we renamed the new app to be `artwork` from `new-artwork`. This is why this code isn't running in production currently (there is no `artwork` page type).